core/shared/src/main/scala/scalaz/zio/ZSchedule.scala

```diff
@@ -292,8 +292,14 @@ trait ZSchedule[-R, -A, +B] extends Serializable { self =>
   /**
    * Returns a new schedule that maps this schedule to a Unit output.
    */
+  @deprecated("use unit", "1.0.0")
   final def void: ZSchedule[R, A, Unit] = const(())
 
+  /**
+    * Returns a new schedule that maps this schedule to a Unit output.
+    */
+  final def unit: ZSchedule[R, A, Unit] = const(())
```

Comment:
Can you make this one delegate to the other one, to avoid duplicated code?

Reply:
Removed duplicate code by delegating to new function.

Reply:
This one is still duplicated!

Reply:
@jdegoes , Somehow git on this link(https://github.com/scalaz/scalaz-zio/pull/742/files/7b0cf4dfb17ee5251e15ee6acdf865eb736e1984#diff-79f04e20de8998f66d7828a15a553ca3
) not showing correct changes after removing duplication.
Please see : https://github.com/scalaz/scalaz-zio/pull/742/files#diff-79f04e20de8998f66d7828a15a553ca3, with duplicate code removal.